packages/relay/src/litepub.ts

```diff
@@ -0,0 +1,297 @@
+              follower,
+              new Accept({
+                id: new URL(`#accepts`, relayActorUri),
+                actor: relayActorUri,
+                object: follow,
+              }),
+            );
+
+            // Send reciprocal follow
+            await ctx.sendActivity(
+              { identifier: RELAY_SERVER_ACTOR },
+              follower,
+              new Follow({
+                actor: relayActorUri,
+                object: follower.id,
+                to: follower.id,
+              }),
+            );
+          } else {
+            await ctx.sendActivity(
+              { identifier: RELAY_SERVER_ACTOR },
+              follower,
+              new Reject({
+                id: new URL(`#rejects`, relayActorUri),
+                actor: relayActorUri,
+                object: follow,
+              }),
+            );
+          }
+        })
+        .on(Accept, async (ctx, accept) => {
+          // Validate follow activity from accept activity
+          const follow = await accept.getObject({
+            crossOrigin: "trust",
+            ...ctx,
+          });
+          if (!(follow instanceof Follow)) return;
+          const follower = follow.actorId;
+          if (follower == null) return;
+
+          // Validate following - accept activity sender
+          const following = await accept.getActor();
+          if (!isActor(following) || !following.id) return;
+          const parsed = ctx.parseUri(follower);
+          if (parsed == null || parsed.type !== "actor") return;
+
+          // Get follower from kv store
+          const followerData = await ctx.data.kv.get([
+            "follower",
+            following.id.href,
+          ]);
+          if (followerData == null) return;
+
+          // Update follower state
+          const updatedFollowerData = { ...followerData, state: "accepted" };
+          await ctx.data.kv.set(
+            ["follower", following.id.href],
+            updatedFollowerData,
+          );
+
+          // Update followers list
+          const followers = await ctx.data.kv.get<string[]>(["followers"]) ??
+            [];
+          followers.push(following.id.href);
+          await ctx.data.kv.set(["followers"], followers);
+        })
+        .on(Undo, async (ctx, undo) => {
+          const activity = await undo.getObject({
+            crossOrigin: "trust",
+            ...ctx,
+          });
+          if (activity instanceof Follow) {
+            if (
+              activity.id == null ||
+              activity.actorId == null
+            ) return;
+            const followers = await ctx.data.kv.get<string[]>(["followers"]) ??
+              []; // actor ids
+
+            const updatedFollowers = followers.filter((id) =>
+              id !== activity.actorId?.href
+            );
+            await ctx.data.kv.set(["followers"], updatedFollowers);
+            ctx.data.kv.delete(["follower", activity.actorId?.href]);
```

Comment:
I guess you've omitted keyword `await` here?


```suggestion
            await ctx.data.kv.delete(["follower", activity.actorId?.href]);
```